### ISSUE TITLE
hidden fields in the ticket

### ISSUE THREAD

**Issue by Sicoval31** [pluginsGLPI/formcreator#3534]
Hi, 
in a forms, some fields are used to have access to other ones, the answer is not needed in a ticket (as information fields).
Why not add an option for each field to hide it in the ticket.
And so ... not add field in the ticket whom no answer as been set.

----
Dans les formulaires, certaines questions servent à afficher des questions masquées.
Avoir une option pour cacher cette question dans le ticket serait intéressante car elle n'est pas utile.
De même, pouvoir ne pas inclure dans le ticket les questions sans réponses serait un plus et allégerai le ticket (surtout avec # FULLFORM)
Exple : 
Q1 : compte utilisateur connu  (O/N)
si oui : affiche Q2 : utilisateur (acteur GLPI)
si non : affiche Q3 : utilisateur (champ texte de saisie libre)

La réponse à Q1 n'est pas utile dans le ticket

**Comment by btry** [comment]
Hi

Formcreator does not has logic to conditionnally show answxers in the targets. It can only hide unanswered questions with the tag ##FULLFORM##.

This would requires a more powerful templating system. 

The plugin reached end of life then this feature will not be added. You may ask for it after the release of GLPI 11 which will implement native forms. They will replace Formcreator.

**Comment by Sicoval31** [comment]
OK, mais :  It can only hide unanswered questions with the tag ##FULLFORM##.
ne fonctionne pas !PAs de réponses aux questions et malgré la balise ##FULLFORM## la question s'affiche bien dans le ticket
![image](https://github.com/user-attachments/assets/0072d180-aa87-4386-b091-2e917ae7bb5a)

On verra en V11 ;)

**Comment by btry** [comment]
Yes: I forgot to mention that unanswered questions are also those which are hidden by a condition. If the question is visible, the answer (even empty) will be visible in  the target.
